src/arb/Complex.jl

```diff
@@ -695,6 +695,23 @@ function rsqrt(x::ComplexFieldElem, prec::Int = precision(Balls))
    return z
 end
 
+@doc raw"""
+    root(x::ComplexFieldElem, n::Int)
+
+Return the principal $n$-th root of $x$.
```

Comment:
What is "the principal $n$-th root of $x$" ? Is this documented anywhere? (Of course we already use the exact same language for `root(a::QQBarFieldElem, n::Int)`

Reply:
I think this is "standard" language for complex analysis and refers to defining exponentiation using the principal logarithm.